src/engine/Eval_generic.ml

```diff
@@ -45,6 +45,8 @@ type value =
   | Float of float
   | String of string (* string without the enclosing '"' *)
   | List of value list
+  | Date of int * int * int
```

Comment:
I think it would be simpler to not introduce an extra value and just use Float.
Unix.gettimeofday returns simply a float representing the number of seconds since a special date in 1970.
Then you can just use the regular < to compare dates.
so today() could internally return a float, and strptime() would still parse the string, but convert the date to a float.